api/controllers/models.py

```diff
@@ -208,7 +269,8 @@ def do_upload_via_train_files(credentials, tid, model_name):
                 cc_contact="dynabench-site@mlcommons.org",
                 template_name="model_train_failed.txt",
                 msg_dict={"name": model_name},
-                subject=f"Model {model_name} training failed. You must include a training file for all classes",
+                subject=f"""Model {model_name} training failed. You must include
```

Comment:
Any specific reason to make this change?

Reply:
Does not let the pre-commit pass without this